packages/bpk-docs/src/components/DocsPageBuilder/TokenSwitcher.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import BpkHorizontalNav, { BpkHorizontalNavItem } from 'bpk-component-horizontal-nav';
+import { BpkTable, BpkTableHead, BpkTableBody, BpkTableRow, BpkTableHeadCell, BpkTableCell } from 'bpk-component-table';
+
+import { formatTokenName, formatTokenValue } from './../../helpers/tokens-helper';
+
+const platforms = {
+  web: {
+    id: 'web',
+    name: 'Web',
+  },
+  ios: {
+    id: 'ios',
+    name: 'iOS',
+  },
+  android: {
+    id: 'android',
+    name: 'Android',
+  },
+};
+
+class TokenSwitcher extends Component {
+  constructor() {
+    super();
```

Comment:
Just a minor thing, but we should accept props in the constructor signature and pass it to `super`

```
constructor(props) {
  super(props);
```

Reply:
Technically you only need to pass props to super if you intend to access `this.props` in the constructor.

However, I reckon you probably just want us to be consistent, in that case I reckon we should make that change across the board in a separate PR - currently we only pass props to super when required.

Reply:
Make sense, cheers